endpoints/tickets/src/main.cpp

```diff
@@ -252,8 +254,34 @@ namespace
 
 				constraint_iter = _args.find("seconds-until-expiration");
 				if (constraint_iter != std::end(_args)) {
-					// TODO Not yet supported by the ticket administration library.
-					logging::warn("{}: Ignoring [seconds-until-expiration]. Not implemented at this time.", fn);
+					// TODO(???): This is equivalent to what the ticket administration library would provide.
+					const auto secs = std::stoll(constraint_iter->second);
+					if (secs < 0) {
+						logging::error(
+							"{}: Invalid value for [seconds-until-expiration] parameter. Must be greater than 0.", fn);
+						return _sess_ptr->send(irods::http::fail(res, http::status::bad_request));
+					}
```

Comment:
The condition checks for `< 0` but the message states "greater than 0". What about `== 0`?

Reply:
Good catch. Will fix that right up.

Reply:
Done.